kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java

```diff
@@ -317,8 +323,8 @@ public void put(Collection<SinkRecord> records) {
         tableRows.add(getRecordRow(record));
       }
 
-      buffer.buffer(tableRows);
-      if (buffer.hasExcess()) {
+      buffer.addAll(tableRows);
+      if (bufferFull(buffer.size())) {
         pauseAllPartitions(tableIdsToTopics.get(table));
```

Comment:
Can you add a log.info() in `pauseAllPartitions` that says which partitions are being paused?